demisto_sdk/commands/common/errors.py

```diff
@@ -2109,4 +2109,4 @@ def invalid_yml_file(error):
     @staticmethod
     @error_code_decorator
     def changed_pack_name(original_name, new_name):
-        return f'The pack name cannot be change. change the pack name from {new_name} to {original_name}.'
+        return f'Pack names cannot be changed. Please rename it back to {original_name} under pack_metadata.'
```

Comment:
This string  is incorrect (my bad), please do not merge until we decide on it [here](https://panw-global.slack.com/archives/C02SU1HJZC3/p1641400316000100)